core/server-entry.ts

```diff
@@ -68,8 +68,9 @@ export default async context => {
     router.onReady(() => {
       if (config.storeViews.multistore === true) {
         let storeCode = context.vs.storeCode // this is from http header or env variable
-        if (router.currentRoute) { // this is from url
-          storeCode = storeCodeFromRoute(router.currentRoute)
+        if (storeCode === undefined && router.currentRoute) { // this is from url
+          const currentRoute = Object.assign({}, router.currentRoute, {host: context.server.request.headers.host})
```

Comment:
I meant to get the `context.server.vs.storeCode` (https://github.com/DivanteLtd/vue-storefront/blob/4c150446cbb66f87d353495bb4cbe41c6dc7d611/core/scripts/server.js#L163) into consideration in heare

Reply:
that header is not really needed anymore but I've added support once again just to keep bc

Reply:
Thanks